code/modules/awaymissions/bluespaceartillery.dm

```diff
@@ -26,7 +26,7 @@
 	dat += "Locked on<BR>"
 	dat += "<B>Charge progress: [reload]/180:</B><BR>"
 	dat += "<A href='byond://?src=\ref[src];fire=1'>Open Fire</A><BR>"
-	dat += "Deployment of weapon authorized by <br>[company_name] Naval Command<br><br>Remember, friendly fire is grounds for termination of your contract and life.<HR>"
+	dat += "Deployment of weapon authorized by <br>[company_name] High Command<br><br>Remember, friendly fire is grounds for termination of your contract and life.<HR>"

```

Comment:
I dont really care about that change.
But according to this, Nanotrasen has a Navy and a Chief Naval Director:
https://wiki.aurorastation.org/index.php?title=File:NT!!_-_Copy.png